src/Raven.Client/Documents/Smuggler/DatabaseSmuggler.cs

```diff
@@ -103,6 +103,14 @@ public Task<Operation> ExportAsync(DatabaseSmugglerExportOptions options, string
             }, tcs.Task, token);
         }
 
+        public Task<Operation> ExportAsync(DatabaseSmugglerExportOptions options, Stream toStream, CancellationToken token = default)
+        {
+            return ExportAsync(options, async stream =>
```

Comment:
what I think you need to use is ExportToStreamAsync method with proper TCS handling